controller/image.yaml

```diff
@@ -40,5 +40,5 @@ run:
           - "standalone-full.xml"
 osbs:
       repository:
-          name: rhba-7-controller-docker
-          branch: jb-controller70
+          name: containers/rhba7-controller
+          branch: jb-rhba7.0-controller-rhel-7
```

Comment:
You missed dash after rhba.

Reply:
Okay, I think I've fixed it. Had to do it in kiesever, controller, and smartrouter.

Reply:
@goldmann Better?